src/libsyncengine/jobs/network/upload_session/uploadsession.cpp

```diff
@@ -128,7 +128,7 @@ void UploadSession::runJob() {
 void UploadSession::uploadChunkCallback(UniqueId jobId) {
     const std::scoped_lock lock(_mutex);
     auto jobInfo = _ongoingChunkJobs.extract(jobId);
-    if (jobInfo.mapped()) {
+    if (!jobInfo.empty()) {
         if (jobInfo.mapped()->hasHttpError() || jobInfo.mapped()->exitCode() != ExitCodeOk) {
```

Comment:
Can `jobInfo.mapped()` be null at this point?

Reply:
Fixed